scripts/qa_crowbarsetup.sh

```diff
@@ -273,11 +273,23 @@ function add_mount()
     local nfssrc="$2"
     local targetdir="$3"
     local zypper_alias="$4"
-    if [ -n "${localreposdir_target}" ]; then
-        [ -n "${bindsrc}" ] && add_bind_mount "${localreposdir_target}/${bindsrc}" "${targetdir}"
+
+    if [ -n "$localreposdir_target" ]; then
+        if [ -z "$bindsrc" ]; then
+            complain 50 "BUG: add_mount() called with empty bindsrc parameter" \
```

Comment:
As mentioned in https://github.com/SUSE-Cloud/automation/pull/226#issuecomment-75439700 it introduces an unwanted and fairly arbitrary indent to subsequent lines.